web-client-classic/public/stylesheets/grnsight.styl

```
@@ -618,6 +618,21 @@ path.mousezone
 .export-excel-workbook-sheet-option-subheader
   margin-left: -20px
 
+.export-radio-group li
+  display: flex;
+  align-items: center;
```

Per UA (user acceptance) feedback, let’s change this to `align-items: flex-start`

```suggestion
  align-items: flex-start;
```

This change usually has a side-effect where the radio will then look off when the item is just one line, so…